http/src/main/scala/org/broadinstitute/dsde/workbench/leonardo/http/service/DiskServiceInterp.scala

```diff
@@ -173,6 +173,14 @@ class DiskServiceInterp[F[_]: Parallel](config: PersistentDiskConfig,
   ): F[GetPersistentDiskResponse] =
     for {
       ctx <- as.ask
+
+      // throw 403 if no project-level permission
```

Comment:
I am always conflicted between returning a 403, or a 404. On one end, returning a 404 is safer, but a 403 is a lot more user friendly. I think I am fine with it be we might want to have a bigger conversation as a team and harmonize the error code we want to return